src/main/java/technology/tabula/detectors/StringSearch.java

```diff
@@ -0,0 +1,691 @@
+package technology.tabula.detectors;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.cli.ParseException;
```

Comment:
No need for `org.apache.commons.cli.ParseException`. None of the methods in this class actually throw that exception, despite their signature.